lib/database/adapters/LhcPeriodStatisticsAdapter.js

```diff
@@ -32,7 +32,13 @@ class LhcPeriodStatisticsAdapter {
     toEntity(databaseObject) {
         const { id, avgCenterOfMassEnergy, lhcPeriod } = databaseObject;
         const distinctEnergies = databaseObject.get('distinctEnergies');
-        const entity = { id, avgCenterOfMassEnergy, distinctEnergies: distinctEnergies?.split(',').map((energy) => Number(energy)) ?? [] };
+        const beamType = databaseObject.get('beamType');
```

Comment:
Why using `.get` ?

Reply:
the same reason, why use get for 'distinctEnergies'
it is not filed defined in model, so it is not accessible by simple object destruction

Reply:
But I suppose you can use `.beamType` no?

Reply:
no

Reply:
as I said object destruction doesn't work, which is shorthand for what you wrote